src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
-import './App.scss';
+import React, { useEffect, useState } from 'react';
 import { MoviesList } from './components/MoviesList';
+import { Movie } from './types/Movie';
+
 import moviesFromServer from './api/movies.json';
+import './App.scss';
+
+const singleCaseQuery = (query: string) => {
+  return query.toLowerCase();
+};
 
 export const App: React.FC = () => {
```

Comment:
I guess, you still could have applied `memo` here as well

Reply:
I will try to do this 👌